cmd/docgen/run.go

```diff
@@ -91,10 +91,10 @@ func eventsForAction(actionJSON []byte) (json.RawMessage, error) {
 		return nil, err
 	}
 
-	events := session.Events()
-	eventJSON := make([]json.RawMessage, len(events))
-	for i, event := range events {
-		typed, err := utils.EnvelopeFromTyped(event)
+	eventLog := session.EventLog()
```

Comment:
Could be OutputLog, but Output having output with output logs? That's a bit nutty. Event log at least removes confusion with top level Output.